.travis.yml

```diff
@@ -3,27 +3,24 @@ language: python
 python:
   - 3.6
   - 3.5
-  - 2.7
 cache: pip
 env:
   global:
     - PYTHONWARNINGS=default,ignore::PendingDeprecationWarning,ignore::ResourceWarning
     - DJANGO_DATABASE_USER_POSTGRES=postgres
     - DJANGO_DATABASE_USER_MYSQL=travis
   matrix:
-    - DJANGO='>=2.1a,<2.2'
+    - DJANGO='>=2.1,<2.2'
     - DJANGO='>=2.0,<2.1'
     - DJANGO='>=1.11,<1.12'
-    - DJANGO='>=1.10,<1.11'
-    - DJANGO='>=1.9,<1.10'
-    - DJANGO='>=1.8,<1.9'
 matrix:
   fast_finish: true
-  exclude:
-    - python: 2.7
-      env: DJANGO='>=2.0,<2.1'
-    - python: 2.7
-      env: DJANGO='>=2.1a,<2.2'
+  include:
+    - { python: "2.7", env: DJANGO='>=1.11,<1.12' }
+
+    - { python: "3.7", env: DJANGO='>=1.11,<1.12', dist: xenial, sudo: true }
```

Comment:
Hm, this installs Django 2.1.x; I'll try it with double quotes instead of single quotes again